app/services/incoming_text_message_service.rb

```diff
@@ -51,6 +51,8 @@ def self.process(params)
         documents: documents
       )
 
+      IntercomService.create_intercom_message_from_sms(contact_record, fw_from_hub: true) if (client.tax_returns.pluck(:status).map(&:to_sym) & TaxReturnStatus::FORWARD_TO_INTERCOM_STATUSES).any?
```

Comment:
or even `client.forward_messages_to_intercom?` convenience method just to ensure the two incoming messages implementations stay in sync.

Reply:
Oh yeah it makes sense to link them together